src/spyglass/settings.py

```diff
@@ -573,6 +580,7 @@ def dlc_output_dir(self) -> str:
     sorting_dir = sg_config.sorting_dir
     waveforms_dir = sg_config.waveforms_dir
     video_dir = sg_config.video_dir
+    export_dir = sg_config.export_dir
```

Comment:
Minor comment: Will this require adding a new directory for the HHMI image?

Reply:
That's a good point - added #908